ern-api-gen/resources/swift/libraries/ern/api.mustache

```diff
@@ -1,184 +1,171 @@
 #if swift(>=4.0)
-@objcMembers public class {{classname}}: NSObject  {
-    {{#operations}}
-    {{#operation}}
-    {{#isEvent}}
-    static let kEvent{{{camelizedNickName}}} = "{{package}}.event.{{operationId}}";
-    {{/isEvent}}
-
-    {{^isEvent}}
-    static let kRequest{{{camelizedNickName}}} = "{{package}}.request.{{operationId}}";
-    {{/isEvent}}
-    {{/operation}}
-    {{/operations}}
-
-    {{#hasEvent}}
+@objcMembers public class {{classname}}: NSObject {
+{{#operations}}
+{{#operation}}
+{{#isEvent}}
+    static let kEvent{{{camelizedNickName}}} = "{{package}}.event.{{operationId}}"
+{{/isEvent}}
+{{^isEvent}}
+    static let kRequest{{{camelizedNickName}}} = "{{package}}.request.{{operationId}}"
+{{/isEvent}}
+{{/operation}}
+{{/operations}}
+{{#hasEvent}}
+
     public lazy var events: {{classname}}Events = {
-        return {{baseName}}Events()
+        {{baseName}}Events()
     }()
-
-    {{/hasEvent}}
+{{/hasEvent}}
 
     public lazy var requests: {{classname}}Requests = {
-        return {{baseName}}Requests()
+        {{baseName}}Requests()
     }()
 }
 
 {{#hasEvent}}
 @objcMembers public class {{classname}}Events: NSObject {
-    {{#operations}}
-    {{#operation}}
-    {{#isEvent}}
-    public func add{{camelizedNickName}}EventListener(eventListener: @escaping ElectrodeBridgeEventListener) -> UUID?{
+{{#operations}}
+{{#operation}}
+{{#isEvent}}
+    public func add{{camelizedNickName}}EventListener(eventListener _: @escaping ElectrodeBridgeEventListener) -> UUID? {
         assertionFailure("should override")
         return UUID()
     }
-    {{/isEvent}}
-    {{/operation}}
 
-    {{#operation}}
-    {{#isEvent}}
-    public func remove{{camelizedNickName}}EventListener(uuid: UUID) -> ElectrodeBridgeEventListener? {
+    public func remove{{camelizedNickName}}EventListener(uuid _: UUID) -> ElectrodeBridgeEventListener? {
         assertionFailure("should override")
         return nil
     }
-    {{/isEvent}}
-    {{/operation}}
 
-    {{#operation}}
-    {{#isEvent}}
```

Comment:
@belemaire - It's a bit hard to see, but this file had three duplicated sections that were consolidated:

```
{{#operation}}
{{#isEvent}}
item 1
{{/isEvent}}
{{/operation}}

{{#operation}}
{{#isEvent}}
item 2
{{/isEvent}}
{{/operation}}

{{#operation}}
{{#isEvent}}
item 3
{{/isEvent}}
{{/operation}}
```

replaced with:

```
{{#operation}}
{{#isEvent}}
item 1

item 2

item 3
{{/isEvent}}
{{/operation}}
```

Not sure what the initial reason was, but does the change look OK to you?

Reply:
:( Yes this looks much better and will indeed produce same output